auth0/rules/Initial FHIR Authorize.js

```diff
@@ -4,18 +4,6 @@ function (user, context, callback) {
     //If we're on a token refresh, or a callback from the patient picker we don't need this rule.
     return callback(null, user, context);
   }
-
-  console.log('Requested aud:' + context.request.query.aud);
```

Comment:
we remove this section here because Inferno does not pass an aud and neither is there a good way to do this at this point. We can check the AUD later during token proxying.

Reply:
Hmm.... the aud parameter is a required /authorize parameter in the smart launch framework.  So this really should be a required check.

Reply:
The aud parameter is checked in the AUTHZ library when checking scopes before querying for resources. exiting so early in the process is currently failing some tests.